<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FileSearch</class>
    <widget class="QWidget" name="FileSearch">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>FileSearch</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget#FileSearch
                {
                background-color:white
                }
            </string>
        </property>
        <widget class="QLineEdit" name="lineEdit_path">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>500</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="placeholderText">
                <string>请输入搜索路径</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_choose">
            <property name="geometry">
                <rect>
                    <x>500</x>
                    <y>0</y>
                    <width>64</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_choose
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_choose:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_choose:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>选择</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_key">
            <property name="geometry">
                <rect>
                    <x>564</x>
                    <y>0</y>
                    <width>372</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="placeholderText">
                <string>请输入关键词</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_start">
            <property name="geometry">
                <rect>
                    <x>936</x>
                    <y>0</y>
                    <width>64</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_start
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_start:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_start:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>开始</string>
            </property>
        </widget>
        <widget class="QListWidget" name="listWidget_result">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>32</y>
                    <width>1000</width>
                    <height>586</height>
                </rect>
            </property>
            <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="styleSheet">
                <string notr="true">QListWidget#listWidget_result
                    {
                    border:none
                    }
                </string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
